tests/Bridge/Symfony/SymfonyAdapterTest.php

```diff
@@ -58,24 +67,21 @@ public function test an active session is retrieved()
     {
         $kernel = $this->createKernel();
         $kernel->boot();
-
         $adapter = new SymfonyAdapter($kernel);
-        $symfonyResponse = $adapter->handle(
+
+        $response = $adapter->handle(
             new ServerRequest(
                 [],
                 [],
-                '/bar',
+                '/foo',
                 null,
                 'php://input',
                 [],
                 [\session_name() => 'SESSIONID']
             )
         );
 
-        self::assertSame(
-            sprintf("%s=SESSIONID; path=/; httponly; samesite=lax", \session_name()),
-            $symfonyResponse->getHeaders()['Set-Cookie'][0]
-        );
```

Comment:
As the cookie is created based on the session cookie params this test will need to be updated. I have loosened this test to only a `Set-Cookie` being present, where to be honest it would be best to ensure that the contents matches. I will update this test 👍